languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -135267,6 +135267,55 @@ The accident victim died from her injuries.
     <!-- Semantic -->
     <!-- ====================================================================== -->
     <category id="SEMANTICS" name="Semantics" type="inconsistency">
+        <rule id="RIGHT_IN_MY_ALLEY" name="right in (up) my alley" default="temp_off">
+            <pattern>
+                <token>right</token>
+                <marker>
+                    <token>in</token>
+                </marker>
+                <token>my</token>
+                <token>alley</token>
+            </pattern>
+            <message>Did you mean 'up'?</message>
+            <suggestion>up</suggestion>
+            <example correction="up">That is right <marker>in</marker> my alley.</example>
+        </rule>
+        <rule id="PEAS_IN_A_POT" name="two peas in a pot (pod)" default="temp_off">
+            <pattern>
+                <token>two</token>
+                <token>peas</token>
+                <token>in</token>
+                <token>a</token>
+                <marker>
+                    <token>pot</token>
+                </marker>
+            </pattern>
+            <message>Did you mean 'pod'?</message>
+            <suggestion>pod</suggestion>
+            <example correction="pod">You are like two peas in a <marker>pot</marker>.</example>
+        </rule>
+        <rule id="TENDER_HOOKS" name="tender hooks (tenterhooks)" default="temp_off">
```

Comment:
I've never heard of 'tenterhooks' until now, so I've learned something new today!